csharp/test/Apache.Arrow.Flight.TestWeb/FlightHolder.cs

```diff
@@ -56,7 +56,7 @@ public FlightInfo GetFlightInfo()
                 new FlightEndpoint(new FlightTicket(_flightDescriptor.Paths.FirstOrDefault()), new List<FlightLocation>(){
                     new FlightLocation(_location)
                 })
-            });
+            }, 2, 4);
```

Comment:
I don't really know how this test works but is this verified anywhere?  Otherwise it seems you might as well use the defaults. Also, should you do `totalRecords: 2, totalBytes: 4` for future readability (I don't really know what the convention is in the C# implementation so feel free to ignore)?

Reply:
There is an existing test that verifies all of the properties, but it's not obvious unless you dig into it.  The test has a few assumptions baked into test flight server that is spun up on startup that make it hard to inject an expected amount.  

However, I was able enhanced the test server to calculate the batch length and total bytes.  I've also added an additional test to ensure those values match what is expected for those fields.